scripts/assets/test_file_exemptions.textproto

```diff
@@ -666,3 +669,4 @@ exempted_file_path: "utility/src/main/java/org/oppia/android/util/threading/Back
 exempted_file_path: "utility/src/main/java/org/oppia/android/util/threading/BlockingDispatcher.kt"
 exempted_file_path: "utility/src/main/java/org/oppia/android/util/threading/ConcurrentCollections.kt"
 exempted_file_path: "utility/src/main/java/org/oppia/android/util/threading/DispatcherModule.kt"
+exempted_file_path: "utility/src/test/java/org/oppia/android/util/utility/NetworkConnectionTestUtil.kt"
```

Comment:
We can write a test for this file. Testing if the function is setting params correctly or not.

Reply:
Can you provide an example of a test case or maybe provide any reference? I'm not able to figure out how to test this. I was thinking of checking if the `ConnectionStatus` is the one that we are setting using `NetworkConnectionTestUtil`. But this test was almost exactly similar to the `ProdNetworkConnectionUtilTest`.

Reply:
I have written the tests with what I said in the above comment. PTAL. Thanks.